lecture_4/03_bet_acceptance_api/cmd/bootstrap/api.go

```diff
@@ -0,0 +1,47 @@
+package bootstrap
+
+import (
+	"github.com/koines/code-cadets-2021/lecture_4/03_bet_acceptance_api/cmd/config"
+	"github.com/koines/code-cadets-2021/lecture_4/03_bet_acceptance_api/internal/api"
+	"github.com/koines/code-cadets-2021/lecture_4/03_bet_acceptance_api/internal/api/controllers"
+	"github.com/koines/code-cadets-2021/lecture_4/03_bet_acceptance_api/internal/api/controllers/validators"
+	"github.com/koines/code-cadets-2021/lecture_4/03_bet_acceptance_api/internal/domain/services"
+	"github.com/koines/code-cadets-2021/lecture_4/03_bet_acceptance_api/internal/infrastructure/rabbitmq"
+	"github.com/streadway/amqp"
+)
+
+const coefficientBound = 10.0
+const paymentLowerBound = 2.0
+const paymentUpperBound = 100.0
```

Comment:
These shouldn't be constants, these should be configurable (via `config.Cfg` object).